blob/driver/driver.go

```diff
@@ -76,6 +79,12 @@ type ReaderAttributes struct {
 type Attributes struct {
 	// ContentType is the MIME type of the blob object. It must not be empty.
 	ContentType string
+	// Metadata holds key/value pairs associated with the blob.
```

Comment:
Since we enforce lowercase in the concrete type. Should the comment here say that the implementation can expect to receive only lowercase attributes?

Reply:
No, these Attributes are going in the other direction -- the driver implementation is creating them and passing them back to the concrete type, which is then returning them to the user. The comment I added in the last PR added more detail about that; I think it's pretty clear now (it was harder before when Attributes was used both in driver and in blob).